docs/business-apps/power-automate/sharepoint-connector-actions-triggers.md

```diff
@@ -4,237 +4,205 @@ ms.date: 03/13/2020
+Constructs a SharePoint REST API to invoke.For more information about this action, visit the following documentation: [Working with the SharePoint Send HTTP Request flow action in Power Automate](../power-automate/guidance/working-with-send-sp-http-request.md).
 
-Constructs a SharePoint REST API to invoke. **Note**: This action may execute any SharePoint REST API you have access to. Proceed with caution.
+> **Note**: This action may execute any SharePoint REST API you have access to. Proceed with caution. 
 
 ### Set content approval status
-
 Sets the content approval status for an item in a list or library that has content approval turned on. You must provide an ETag for pages and files. You can get the ETag using the Get File Metadata action. This action is only available for SharePoint Online and SharePoint 2019.
 
 ### Set hub site join status to pending
-
 Sets the requested site's hub join request status to pending. The Approval Correlation Id is an optional parameter that helps SharePoint identify a particular hub join request. The requesting site can only have one pending request at a given time.
 
 ### Stop sharing an item or a file
-
 Deletes all links giving access to an item or a file, and removes all people with direct access, except for owners.
 
 ### Update file
-
 Updates the contents of the file specified by the file identifier.
 
 ### Update file properties
-
 Updates the properties stored in columns in a library for the item specified by the item ID. To update file contents, use the **Update file** action. When using this with the On-Premises Data Gateway, you may need to manually enter the name of the library to connect to.
 
 ### Update item
-
 Updates an item in a SharePoint list.
+
+## Known limitations
+### Supported list and library templates
+Power Automate flows for lists are only supported in generic lists (100) and generic document libraries (101). Custom list and library templates are currently not supported; including but not limited to lists such as Announcements, Contacts and Tasks.
+
+### Flow runs
+When a Power Automate flow is built to be triggered for an item or a file creation and/or modification, Power Automate periodically checks for changes in the list or library, configured in the flow trigger. In most cases, if there is a single change in the list or library, the flow run may occur within minutes after that change. However, it is expected that the flow may batch more than one change in subsequent flow runs when the flow interval is large since its last valid change or subsequent edits to an item or a file.
+
+### File move and flow runs
+When users move one or more files from one document library to another, the original file is moved from the source library to the destination library. Moving the file will not alter any custom metadata including when the file was created and modified. Hence, this action will not trigger any flows for those file updates associated in the library where it was moved. 
```

Comment:
When you move one or more files from one document library to another, the original file is moved from the source library to the destination library. Moving the file does not alter any custom metadata, including when the file was created and modified. Hence, this action will not trigger any flows for those file updates associated in the library where it was moved.